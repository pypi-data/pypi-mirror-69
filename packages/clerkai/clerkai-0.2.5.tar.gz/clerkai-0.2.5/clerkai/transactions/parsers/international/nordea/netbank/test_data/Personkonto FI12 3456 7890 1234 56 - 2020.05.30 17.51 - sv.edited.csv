Bokföringsdag;Belopp;Avsändare;Mottagare;Namn;Rubrik;Saldo;Valuta
07.05.2020;-72,00;NAID-FI-EUR-FI3814703500919488;;;PAYBYWAY OY;;EUR
30.12.2019;-4,16;NAID-FI-EUR-FI3814703500919488;;;S MARKET FOO;;EUR