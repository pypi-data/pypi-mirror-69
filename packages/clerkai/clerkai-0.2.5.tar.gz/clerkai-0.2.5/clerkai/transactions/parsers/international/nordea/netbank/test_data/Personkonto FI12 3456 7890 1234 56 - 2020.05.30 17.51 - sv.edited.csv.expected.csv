Real Date,Bank Date,Payer,Payee,Bank Message,Amount,Balance,Currency,Original data,Raw Real Date,Raw Bank Date,Raw Payer,Raw Payee,Raw Bank Message,Raw Amount,Raw Balance,Raw Currency
,2020-05-07,NAID-FI-EUR-FI3814703500919488,,PAYBYWAY OY,-72.00,,EUR,"{'Bokföringsdag': '07.05.2020', 'Belopp': -72.0, 'Avsändare': 'NAID-FI-EUR-FI3814703500919488', 'Mottagare': nan, 'Namn': nan, 'Rubrik': 'PAYBYWAY OY', 'Saldo': nan, 'Valuta': 'EUR'}",,07.05.2020,NAID-FI-EUR-FI3814703500919488,,PAYBYWAY OY,-72.0,,EUR
,2019-12-30,NAID-FI-EUR-FI3814703500919488,,S MARKET FOO,-4.16,,EUR,"{'Bokföringsdag': '30.12.2019', 'Belopp': -4.16, 'Avsändare': 'NAID-FI-EUR-FI3814703500919488', 'Mottagare': nan, 'Namn': nan, 'Rubrik': 'S MARKET FOO', 'Saldo': nan, 'Valuta': 'EUR'}",,30.12.2019,NAID-FI-EUR-FI3814703500919488,,S MARKET FOO,-4.16,,EUR
